File: Python/secao08/Exercicio02.py
# var 
vetor1 = []
vetor2 = []
soma = []

# processing
for i in range(0, 10):
    num1 = int(input("Informe um valor para o primeiro vetor: "))
    vetor1.append(num1)
    num2 = int(input("Informe um valor para o segundo vetor: "))
    vetor2.append(num2)
    soma.append(num1 + num2)
    
for i in soma:
    print(i)

File: C/Exercices/secao05/exercicio02.c
#include <stdio.h>

int main()
{
    // var
    int vetor1[10], vetor2[10], soma[10];

    for (int i = 0; i < 10; i++)
    {
        printf("Digite um valor para o primeiro vetor: ");
        scanf("%d", &vetor1[i]);
        printf("Digite um valor para o segundo vetor: ");
        scanf("%d", &vetor2[i]);

        soma[i] = (vetor1[i] + vetor2[i]);
    }
    for (int i = 0; i < 10; i++)
    {
        printf("Soma %d: ", i);
        printf("%d\n", soma[i]);
    }
}
